### PR TITLE
Adjustment for removed `with<childName>` functions

### PR DESCRIPTION
**Pull request by ahoppen**
Companion of https://github.com/apple/swift-syntax/pull/1253

### CODE REVIEW

**Comment by ahoppen** [comment]
https://github.com/apple/swift-syntax/pull/1253
https://github.com/apple/swift-format/pull/477

@swift-ci Please test

File: SwiftEvolve/Sources/SwiftEvolve/SyntaxExtensions.swift [hunk 1]
```diff
@@ -19,8 +19,7 @@ import SwiftSyntax
 import Foundation
```
Comment (bnbarham) [inline SwiftEvolve/Sources/SwiftEvolve/SyntaxExtensions.swift:20]: These seem like they could be useful to have in SwiftSyntax itself. Not a blocker, just a thought.

**Review by bnbarham — APPROVED** [review]

File: SwiftEvolve/Sources/SwiftEvolve/SyntaxExtensions.swift (hunk 1 above)
Comment (ahoppen) [reply SwiftEvolve/Sources/SwiftEvolve/SyntaxExtensions.swift:20]: Yeah, that’s true. More traits seem useful in general.

**Comment by ahoppen** [comment]
https://github.com/apple/swift/pull/63160
https://github.com/apple/swift-syntax/pull/1253
https://github.com/apple/swift-format/pull/477

@swift-ci Please test

**Comment by ahoppen** [comment]
https://github.com/apple/swift/pull/63160
https://github.com/apple/swift-syntax/pull/1253
https://github.com/apple/swift-format/pull/477

@swift-ci Please test

**Comment by ahoppen** [comment]
https://github.com/apple/swift/pull/63160
https://github.com/apple/swift-syntax/pull/1253
https://github.com/apple/swift-format/pull/477

@swift-ci Please test

**Pull request merged**